bugbot/rules/variant_expiration.py

```diff
@@ -339,6 +362,15 @@ def handle_bug(self, bug, data):
 
         return bug
 
+    def get_needinfo_id(self, bug: dict) -> str:
+        """Get the ID of the needinfo flag"""
+
+        for flag in bug.get("flags", []):
+            if flag["name"] == "needinfo":
+                return flag["id"]
```

Comment:
This will get all needinfo, but we are interested in removing only needinfos requested by our bot.